prophy/composite.py

```diff
@@ -4,6 +4,7 @@
 from .exception import ProphyError
```

Comment:
Please make sure that tox tests pass. Tox runs py.test tests in python 2.7 and 3.4. Currently the latter fails:
[tox.prophy.txt](https://github.com/aurzenligl/prophy/files/338197/tox.prophy.txt)
[tox.prophy_jiish.txt](https://github.com/aurzenligl/prophy/files/338198/tox.prophy_jiish.txt)